packages/components/checkbox/src/checkbox.js

```diff
@@ -50,7 +50,8 @@ export class TSCheckbox extends TSElement {
 					?disabled="${this.disabled}"
 				/>
 				<div class="checkbox"></div>
-				${this.label}
+				<!-- To customized checkbox label (links, ...) -->
+				<slot> ${this.label} </slot>
```

Comment:
We need to fix the story, because there is a whitespace and label do not render.
also, maybe we should somehow document, that data-label will not work if you have any whitespaces and special characters in your template, e.g. this examples will not render the label:
```html
<ts-checkbox data-label="Label"> </ts-checkbox>

<ts-checkbox data-label="Label">
</ts-checkbox>
```